triggers/service/auth/BUILD.bazel

```diff
@@ -71,6 +71,11 @@ da_scala_library(
         "@maven//:com_typesafe_akka_akka_parsing",
         "@maven//:com_typesafe_akka_akka_stream",
         "@maven//:com_typesafe_scala_logging_scala_logging",
+        "@maven//:com_chuusai_shapeless",
+        "@maven//:com_github_pureconfig_pureconfig_core",
+        "@maven//:com_github_pureconfig_pureconfig_generic",
+        "@maven//:org_typelevel_cats_core",
+        "@maven//:org_typelevel_cats_kernel",
```

Comment:
Ideally I would prefer if we could avoid having both Scalaz and Cats in at the same time. Was the required for Pureconfig?

Reply:
yup good shout , removed cats